core/Db/Schema/Mysql.php

```diff
@@ -472,6 +477,28 @@ public function createAnonymousUser()
                     VALUES ( 'anonymous', '', 'anonymous', 'anonymous@example.org', 'anonymous', 0, '$now', '$now' );");
     }
 
+    /**
+     * Creates an entry in the User table for the "anonymous" user.
+     */
+    public function recordInstallVersion()
+    {
+        if (!self::getInstallVersion()) {
+            Option::set(self::OPTION_NAME_MATOMO_INSTALL_VERSION, Version::VERSION);
+        }
+    }
+
+    /**
+     * Creates an entry in the User table for the "anonymous" user.
+     */
```

Comment:
Think this & the above docblock needs to be changed or removed.

Reply:
👍 will update